mobile/examples/question_answering/android/app/src/main/java/ai/onnxruntime/example/question_answering/MainActivity.kt

```diff
@@ -0,0 +1,99 @@
+package ai.onnxruntime.example.question_answering
+
+import ai.onnxruntime.*
+import ai.onnxruntime.extensions.OrtxPackage
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.util.Log
+import android.widget.Button
+import android.widget.TextView
+import android.widget.Toast
+import androidx.activity.*
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.coroutines.*
+import java.util.*
+
+
+class MainActivity : AppCompatActivity() {
+    private var ortEnv: OrtEnvironment = OrtEnvironment.getEnvironment()
+    private lateinit var ortSession: OrtSession
+    private val TitleText: TextView by lazy {findViewById(R.id.TitleView)}
+    private val ArticleText: TextView by lazy {findViewById(R.id.ArticleView)}
+    private val AnswerText: TextView by lazy {findViewById(R.id.AnswerView)}
+    private val QuestionText: TextView by lazy {findViewById(R.id.QuestionView)}
+    private val DoQAButton: Button by lazy {findViewById(R.id.PerformQaButton)}
+
+    @SuppressLint("UseCompatLoadingForDrawables")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        // from <The Little Prince>, chapter one, and the first paragraph
+        TitleText.setText("<The Little Prince>")
+        ArticleText.setText(
+            "we are introduced to the narrator, a pilot, and his ideas about grown-ups." +
+                    "Once when I was six years old I saw a magnificent picture in a book," +
+                    " called True Stories from Nature, about the primeval forest. " +
+                    "It was a picture of a boa constrictor in the act of swallowing an animal. " +
+                    "Here is a copy of the drawing." +
+                    "In the book it said: \"Boa constrictors swallow their prey whole, " +
+                    "without chewing it. After that they are not able to move," +
+                    " and they sleep through the six months that they need for digestion.\""
+        );
+        QuestionText.setHint("From which book did I see a magnificent picture?")
```

Comment:
not a comment, but assume `setHint` comes from Android `TextView`? wondering if iOS Swift has such equivalent features, seems like no.

that's okay should definitely be able to find out a work around in Swift. Good to see a hint will show when the input text box is null.

Reply:
Yeah, It comes from TextView.

It seems be PlaceHolder in IOS UITextView and It does't have the default method to use it though. I found some material which may work it through.

Reply:
How hard would it be to modify the app so that the user can enter a question and a context?

Reply:
Not too hard. Do we want to support entering context in this example APP?